src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  const PRICE_PER_DAY = 40;
```

Comment:
Just give a name with lowerCase & camelCase
use upperCase when you need a variable in a lot of cases
```suggestion
  const pricePerDay = 40;
```